flink-connectors/flink-connector-cassandra/src/test/java/org/apache/flink/streaming/connectors/cassandra/CassandraConnectorITCase.java

```diff
@@ -382,31 +383,31 @@ public void testCassandraCommitter() throws Exception {
 	@Test
 	public void testCassandraTupleAtLeastOnceSink() throws Exception {
 		CassandraTupleSink<Tuple3<String, Integer, Integer>> sink = new CassandraTupleSink<>(injectTableName(INSERT_DATA_QUERY), builder);
-
-		sink.open(new Configuration());
-
-		for (Tuple3<String, Integer, Integer> value : collection) {
-			sink.send(value);
+		try {
```

Comment:
please limit your changes to your sink, or move these into a separate commit.

Reply:
ok